### PR TITLE
Add support of arduino connector on nucleo_wb55rg board

### PR DESCRIPTION
**Pull request by tisserab**
Support of Arduino connector added on nucleo_wb55rg board.  Thanks in advance for you review and comments.

### CODE REVIEW

File: boards/arm/nucleo_wb55rg/arduino_r3_connector.dtsi [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2019 Linaro Limited
```
Comment (erwango) [inline boards/arm/nucleo_wb55rg/arduino_r3_connector.dtsi]: Please use STMicro copyright here.

File: boards/arm/nucleo_wb55rg/arduino_r3_connector.dtsi [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+			   <18 0 &gpioa 6 0>,	/* D12 */
+			   <19 0 &gpioa 5 0>,	/* D13 */
+			   <20 0 &gpiob 9 0>,	/* D14 */
+			   <21 0 &gpiob 8 0>;	/* D15 */
+	};
+};
+
+arduino_i2c: &i2c1 {};
+arduino_spi: &spi1 {};
+
```
Comment (erwango) [inline boards/arm/nucleo_wb55rg/arduino_r3_connector.dtsi]: Since we're there, can you add arduino_uart ?

**Review by erwango — CHANGES_REQUESTED** [review]
Thanks for this addition, looks good, a couple of (minor) comments though.

File: boards/arm/nucleo_wb55rg/arduino_r3_connector.dtsi (hunk 2 above)
Comment (ulfalizer) [inline boards/arm/nucleo_wb55rg/arduino_r3_connector.dtsi]: Could remove extra blank line at the end of the file.

**Review by erwango — CHANGES_REQUESTED** [review]
Looks globally fine, you just need to squash all these commits into a single one.

File: boards/arm/nucleo_wb55rg/nucleo_wb55rg.yaml [hunk 3]
```diff
@@ -10,10 +10,12 @@ ram: 96
...
 supported:
   - gpio
-  - arduino_i2c
   - i2c
   - counter
   - spi
   - pwm
   - adc
   - watchdog
+  - arduino_serial
```
Comment (erwango) [inline boards/arm/nucleo_wb55rg/nucleo_wb55rg.yaml]: I know this was my request but `arduino_serial` doesn't exists, this should actually be `arduino_gpio`. My bad.
Can you do the change?

**Review by erwango — CHANGES_REQUESTED** [review]

**Review by erwango — APPROVED** [review]

**Review by galak — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**